frontend/src/metabase/embedding-sdk/test/rename-conflicting-cljs-globals.ts

```diff
@@ -0,0 +1,13 @@
+const CONFLICTING_CLJS_GLOBALS = ["cljs", "malli", "metabase"];
+
+// To properly work with CLJS runtime that comes with Embedding SDK bundle
+export function renameConflictingCljsGlobals() {
+  for (const cljsGlobal of CONFLICTING_CLJS_GLOBALS) {
+    if (cljsGlobal in window) {
+      // @ts-expect-error -- dynamic property access
+      window[`${cljsGlobal}_tmp`] = window[cljsGlobal];
+      // @ts-expect-error -- dynamic property access
+      delete window[cljsGlobal];
+    }
+  }
+}
```

Comment:
A hack just for storybook/cypress to deal with conflicting cljs runtimes
Must NOT be used with the production SDK code